src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepDynamicContext.java

```diff
@@ -158,8 +179,8 @@ private static abstract class Translator<T> extends DynamicContext.Typed<T> {
         }
 
         @Override protected FilePath get(DelegatedContext context) throws IOException, InterruptedException {
-            if (LOGGER.isLoggable(Level.FINE)) {
-                LOGGER.fine("ESDC=" + context.get(ExecutorStepDynamicContext.class) + " FPR=" + context.get(FilePathDynamicContext.FilePathRepresentation.class));
+            if (LOGGER.isLoggable(Level.FINER)) {
+                LOGGER.finer("ESDC=" + context.get(ExecutorStepDynamicContext.class) + " FPR=" + context.get(FilePathDynamicContext.FilePathRepresentation.class));
```

Comment:
Moving this down to `FINER` to make it easier to focus on resumption-related messages with a custom logger.

Reply:
😅 https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/344#discussion_r1342890151

Reply:
Sigh, I keep on forgetting that checked exceptions are basically incompatible with all newer language features.